attributes/geo:proj/README.md

```diff
@@ -0,0 +1,183 @@
+# Projection Attribute Extension for Zarr
+
+- **Extension Name**: Projection Attribute Extension
+- **Version**: 1.0.0
+- **Extension Type**: Attribute
+- **Status**: Proposed
+- **Owners**: @emmanuelmathot
+
+## Description
+
+This specification defines a JSON object that encodes coordinate reference system (CRS) information for geospatial data. Additionally, this specification defines a convention in which this object is stored under the `"geo:proj"` key in the attributes of Zarr groups or arrays.
+
+**Recommended usage**: Define `geo:proj` at the **group level** to apply CRS information to all arrays within that group. This matches the common geospatial pattern of storing multiple arrays with the same coordinates in a single group. Array-level definitions are supported for override cases but are less common.
+
+## Motivation
+
+- Provides simple, standardized CRS encoding without complex nested structures
+- Compatible with existing geospatial tools (GDAL, rasterio, pyproj)
+- Based on the proven STAC Projection Extension model
+
+## Inheritance Model
+
+The `geo:proj` attribute follows a simple group-to-array inheritance model that should be understood first:
+
+### Inheritance Rules
+
+1. **Group-level definition** (recommended): When `geo:proj` is defined at the group level, it applies to all arrays within that group
+2. **Array-level override**: An array can completely override the group's `geo:proj` attribute with its own definition
+3. **Complete replacement only**: Partial inheritance (overriding only some fields while inheriting others) is not allowed
+4. **No cascading**: Inheritance only applies from a group directly to its immediate array members, not through nested groups
+
+Most use cases will use group-level definitions without array overrides.
+
+## Specification
+
+The `geo:proj` attribute can be added to Zarr arrays or groups to define projection information.
+
+### Required Fields
+
+At least one of the following MUST be provided:
+
+- `code`: Authority and code identifier (e.g., "EPSG:4326")
+- `wkt2`: WKT2 string representation of the CRS
+- `projjson`: PROJJSON object representation of the CRS
+- `version`: Version of the `geo:proj` extension being used
+
+### Optional Fields
+
+- `bbox`: Bounding box in the CRS coordinates
+- `transform`: Affine transformation coefficients (6 or 9 elements)
+- `spatial_dimensions`: Names of spatial dimensions in the array
```

Comment:
the JSON types of these fields should be given -- is the `bbox` an array of numbers, or an array of arrays of numbers?

Reply:
Would we want to automate that in the github actions or in pre-commit?

Reply:
I like the table! But maybe other people have different opinions